packages/beacon-node/test/unit/network/nodejs/libp2p.test.ts

```diff
@@ -1,59 +0,0 @@
-import {assert} from "chai";
-import {sleep} from "@lodestar/utils";
-import {NodejsNode} from "../../../../src/network/nodejs/index.js";
-import {createNode} from "../../../utils/network.js";
-import {Libp2pEvent} from "../../../../src/constants/index.js";
-
-const multiaddr = "/ip4/127.0.0.1/tcp/0";
-
-describe("[network] nodejs libp2p", () => {
```

Comment:
If this test adds value, and is deleted for rushness please open and issue to add it again latter

Reply:
test doesn't add value. tests upstream libp2p behavior.